mojo/edk/system/broker_win.cc

```diff
@@ -156,5 +180,5 @@ scoped_refptr<PlatformSharedBuffer> Broker::GetSharedBuffer(size_t num_bytes) {
   return nullptr;
 }
 
-}  // namespace edk
+}  // namespace mojo
```

Comment:
It seems namespace mojo is closed two times. Is there no compile error?

Reply:
There are three namespaces in the file. 1. mojo, 2. edk. 3. a empty name space.
3 is closed above as I mentioned.
In this line edk and mojo namespaces are closed as usual.
Some how Visual studio is integrated with the clang-format. Once I close a file auto format is done.
That is the reason many files have formatting changes also.

Reply:
Corrected on latest patch set.